src/main/java/org/diego/tutorial/car/timers/CarsTimer.java

```diff
@@ -0,0 +1,38 @@
+package org.diego.tutorial.car.timers;
+
+import java.util.List;
+
+import javax.ejb.EJB;
+import javax.ejb.Schedule;
+import javax.ejb.Singleton;
+
+import org.apache.log4j.Logger;
+import org.diego.tutorial.car.model.Car;
+import org.diego.tutorial.car.model.service.CarService;
+
+/**
+ * Class that implements a timer for the operations related to the cars
+ * */
+@Singleton
+public class CarsTimer {
```

Comment:
the name of this class should be more self-descriptive

Reply:
done